am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -88,6 +88,25 @@ public static ExplicitAccessMetadata createMetadata(String resourceId, String ac
                 .resourceName(RESOURCE_NAME)
                 .build())
             .attribute(JsonPointer.valueOf(""))
+            .securityClassification(PUBLIC)
```

Comment:
Now we have two `createMetadata` methods. Please make one delegate processing to the other one.

Reply:
👍